packages/shared/src/graphql/posts.ts

```diff
@@ -39,6 +39,16 @@ export const supportedTypesForPrivateSources = [
   PostType.Freeform,
 ];
 
+type PostFlags = {
+  sentAnalyticsReport: boolean;
+  banned: boolean;
+  deleted: boolean;
+  private: boolean;
+  visible: boolean;
+  showOnFeed: boolean;
+  promoteToPublic: number;
```

Comment:
Was it intentional that the type for `promoteToPublic` is a number?

Reply:
Yeah its a unix timestamp